src/components/TransactionDetails/TransactionDetailsTs.ts

```diff
@@ -15,40 +15,11 @@
  */
 import {Component, Prop, Vue} from 'vue-property-decorator'
 import {mapGetters} from 'vuex'
-import {
-  AccountAddressRestrictionTransaction,
-  AccountLinkTransaction,
-  AccountMetadataTransaction,
-  AccountMosaicRestrictionTransaction,
-  AccountOperationRestrictionTransaction,
-  AddressAliasTransaction,
-  AggregateTransaction,
-  HashLockTransaction,
-  MosaicAddressRestrictionTransaction,
-  MosaicAliasTransaction,
-  MosaicDefinitionTransaction,
-  MosaicGlobalRestrictionTransaction,
-  MosaicId,
-  MosaicMetadataTransaction,
-  MosaicSupplyChangeTransaction,
-  MultisigAccountModificationTransaction,
-  NamespaceMetadataTransaction,
-  NamespaceRegistrationTransaction,
-  NetworkType,
-  SecretLockTransaction,
-  SecretProofTransaction,
-  Transaction,
-  TransactionType,
-  TransferTransaction,
-} from 'symbol-sdk'
-
+import {AccountAddressRestrictionTransaction, AccountLinkTransaction, AccountMetadataTransaction, AccountMosaicRestrictionTransaction, AccountOperationRestrictionTransaction, AddressAliasTransaction, AggregateTransaction, HashLockTransaction, MosaicAddressRestrictionTransaction, MosaicAliasTransaction, MosaicDefinitionTransaction, MosaicGlobalRestrictionTransaction, MosaicId, MosaicMetadataTransaction, MosaicSupplyChangeTransaction, MultisigAccountModificationTransaction, NamespaceMetadataTransaction, NamespaceRegistrationTransaction, NetworkType, SecretLockTransaction, SecretProofTransaction, Transaction, TransactionType, TransferTransaction} from 'symbol-sdk'
```

Comment:
This should not pass the linter (120 max-len)

It's better to have them in a column so the unused var are easier to spot

Reply:
We have added prettier with eslint to the ts-sdk. https://github.com/nemtech/symbol-sdk-typescript-javascript/pull/531. Prettier is really good, you may want to consider it for the wallet. I did spend a lot of time trying to replicate the style formatting in Intellij (back in the days in the SDK and last week on the wallet), If I had prettier it would have worked out of the box for any IDE (as it's cli based). It's also integrated with eslint (eslint fails if the code is not 'pretty' enough). Having an IDE independent consistent style is great. eslint --fix only fixes some of the style, not everything (prettier fills the gap). I think prettier can work with Vue, but this needs to be tested out. 

We can integrate prettier after the release and this PR when there aren't too many open PRs